README.md

```diff
@@ -1,51 +1,44 @@
 # Sandbox
 
-This repository collects projects that have followed our pipeline and have been sponsored by the community, they are open source and support our mission. 
+This repository highlights projects that are supported by the OpenData community; these are all open source and support our mission. 
 
 The Sandbox pipeline have been designed by the community and can be found [here on our forum](https://forum.opendatacommunity.org/t/project-pipeline-in-detail/28/5).
 
-To create your project please start through the sandbox teams [repo](https://github.com/orgs/OpenDataforWeb3-sandbox/repositories) and follow the pipeline, as your project matures and fulfills the process requirements it will be displayed here.
+To create your project please start through the sandbox teams [repo](https://github.com/orgs/OpenDataforWeb3-sandbox/repositories) and follow the pipeline, as your project matures and fulfills the process requirements it will be displayed here.  
 
-## Current Sandbox Teams
+## Current Sandbox Projects
 
 1. [Grant Review Dashboard](https://github.com/OpenDataforWeb3-sandbox/Grant-Review-Dashboard)
 1. [Lego Curration](https://github.com/OpenDataforWeb3-sandbox/Lego-Curration)
 1. [Sybil Dashboard](https://github.com/OpenDataforWeb3-sandbox/Sybil-Dashboard)
 
```

Comment:
We also support the decentralized and local indexing project TrueBlocks as a Sandbox project,
Please visit their web site to learn more.  [TrueBlocks.io](https://trueblocks.io/)